pages/Addons.md

```diff
@@ -44,6 +44,8 @@ ExtraTools | Sfiguz | <details><summary>expand</summary>ExtraTools is a Slimefun
 TranscEndence | Sfiguz | <details><summary>expand</summary>This Addon adds tools to get permanent potion effects via a quantum physics-like experience. It was submitted as an entry to the 2020 Slimefun Addon Jam.</details> | N/A | [GitHub](https://github.com/Sfiguz7/TranscEndence) | [Builds](https://thebusybiscuit.github.io/builds/Sfiguz7/TranscEndence/master/)
 SFCalc | Seggan | <details><summary>expand</summary>This addon is a continuation of the old SlimeCalculator by John000708. It provides a command to calculate the required amounts of base resources for any given Slimefun item.</details> | N/A | [GitHub](https://github.com/Seggan/SFCalc) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/SFCalc/master/)
 Slimy Power Suits (WIP) | Seggan | <details><summary>expand</summary>This addon is an expansion for the popular addon LiteXpansion. It adds modular power suits to Slimefun.</details> | N/A | [GitHub](https://github.com/Seggan/Slimy-Power-Suits) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/Slimy-Power-Suits/master/)
+SlimefunWarfare | Seggan | <details><summary>expand</summary>This addon adds guns, chemical grenades, and more war stuff to Slimefun</details> | N/A | [GitHub](https://github.com/Seggan/SlimefunWarfare) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/SlimefunWarfare/master/)
+Liquid | Seggan, NCBPFluffyBear | <details><summary>expand</summary>This addon adds liquid version of Slimefun metals and alloys. It also adds ways to more effectiently make alloys with the molten metals</details> | N/A | [GitHub](https://github.com/Seggan/Liquid) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/Liquid/master/)
```

Comment:
Please add your addons to the bottom of the list and not somewhere in the middle...

Reply:
well thats where the rest of my addons are...

Reply:
So?